integration/tests/upgrade.sh

```diff
@@ -6,16 +6,3 @@ test_upgrades=true
 test_upgrade_inspec_profiles=(a2-deploy-integration)
 # The inspec tests don't pass if the diagnostics are run
 test_skip_diagnostics=true
-
-run_upgrade() {
```

Comment:
why is this not needed now?

Reply:
since this function is there in deployment.sh